### ISSUE TITLE
Model.eval()?

### ISSUE THREAD

**Issue by eambutu** [katerakelly/pytorch-maml#5]
Given that we are using Batch Normalization layers here, shouldn't we be calling model.eval() before getting accuracy numbers on the test set?

**Comment by eambutu** [comment]
Just kidding, I just noticed that momentum = 1.
